src/commands/test.js

```diff
@@ -90,3 +141,109 @@ function getPlatform(logs) {
 
   throw new Error(`Unsupported platform: ${type} ${arch} ${majorVersion}`)
 }
+
+function runDocker(coverage_opt, datasource, version_opt, latest_version, print) {
+  // Remove binary-install-raw binaries, because docker has permission issues
+  // when building the docker images
+  fs.rmSync("node_modules/binary-install-raw/bin", { force: true, recursive: true });
```

Comment:
Could there be a chance that this tries to remove the directory from the wrong place?

Like running the CLI from a diferent directory or something 🤔 

Since the path here is relative.

Reply:
Good point! For now (we should note this somewhere) the docker implementation is meant to be run from the subgraph root folder.

I will look into making it possible to be ran from anywhere.

Reply:
Ok, so basically this is an issue if someone uses the graph-cli that is installed as a dependency in the subgraph's local node_modules, e.g. `demo-subgraph/node_modules/@graphprotocol/graph-cli`, and if you run `demo-subgraph/node_modules/@graphprotocol/graph-cli/bin/graph test` the matchstick binary will be installed in `demo-subgraph/node_modules/binary-install-raw/bin`, and this will be an issue for the docker. So we can either add a .dockerignore file in the root folder of the subgraph, or delete the bin folder, as we do here. If you use the globally installed graph-cli via `npm install -g @graphprotocol/graph-cli`, this won't be an issue.

Reply:
Ok, nice, makes sense 👍